products/mozilla_org/test_mozilla_org.md

```diff
@@ -0,0 +1,67 @@
+* Check for truncation: button text remains inside the button
+* Header line break wraps at proper place
+* Text not overlapping graphic
+* Terminology is consistency with product, and between web pages
+* Brand names remain in English
+* Product names comply to Mozilla guideline and adhere to what the community has agreed to
+* No corrupted characters
+* Click on the links on the page, which should take you to the pages of the same language if they are localized?
+* Nav bar terms consistent with the page titles they are linked to 
+* Footer links don’t overlap with one another
+
+You can make linguistic changes directly in Pontoon, Locamotion or github.  
+
+## Functionality Testing
+* Click the download link, you should be able to download the product in your language, if it is localized, such as Firefox.
+* Font support and readability
+* Footer: links point to pages of in the same language if those pages, sites or documents are localized
+* Language list: your language is listed as one of the option
+* Deliberately type a wrong broken link, verify whether 404 page is in the same language.
+* Sorting order of languages in dropdown list
+* In RTL (right to left languages), both target language and English flows in the correct directions.  The mirroring effect is correct as defined by Mozilla UX team.  Indentation and alignment are correct.
+
+
+## Testing for Fundraising Campaigns: 
+The annual fundraise campaign is localized in many languages, but not all languages. 
+* Date and time format is country specific
+* Currency availability
+* Postal/zip code in the proper format
+* Mailing address format appropriate for the country
+* Payment system available for the targeted country
+* Country list in your language and order
+
+## Compatibility testing:
+* Test the page layout in other major browsers and on other platform
+* Test the page layout on the leading home grown browser if available.
+* Test the page layout on mobile devices of major platforms
+
+## When Can I See the Localized Page on Production? 
+
+Code push to production is made almost daily.  When a brand new page is being localized, the page won't go live until it is fully localized for the very first time. Once a page is on the production, an ongoing maintenance is required, as updates take place on a irregular basis. When it is updated with minor change, especially when a new string is associated with a tag, the page will be updated when the tagged string is localized.  Sometimes a tagged string has a firm deadline.  If the new tagged string is not localized by certain date, the new string will be pushed to production regardless.  If localized content is not kept up, either the page is completely out of date over time with old designs and message, or the site contains mixed of English and target language. We may have to disable it if the latter is the case. Check the team [Webpage dashboard](https://l10n.mozilla-community.org/webdashboard/) for updates.  
```

Comment:
the production -> production, or the production server
webpage dashboard -> web dashboard

I don't think we should explain tags to them, they don't see them. We should just tell them that new strings added to existing pages are visible in production only when localized, unless they're in a shared file.

For the last part, the concept is that sometimes a completely new template is used for a page, e.g. the home page, and that deadlines are strict: after the deadline, the old template will be disabled. All these big changes are communicated well in advance on the mailing list.